dash/development/_py_components_generation.py

```diff
@@ -488,11 +488,12 @@ def shape_or_exact():
                 if js_to_py_type(subType) != '')),
 
         # React's PropTypes.arrayOf
-        arrayOf=lambda: 'list'.format(  # pylint: disable=too-many-format-args
-            ' of {}s'.format(
-                js_to_py_type(type_object['value']))
-            if js_to_py_type(type_object['value']) != ''
-            else ''),
+        arrayOf=lambda: (
```

Comment:
@alexcjohnson Hmm. Now we're seeing 5 unit test failures:

``
tests/unit/dash/development/test_base_component.py:944: in assert_docstring
    ])[i]
E   AssertionError: '- optionalArrayOf (list of numbers; optional)' != '- optionalArrayOf (list; optional)'
E   - - optionalArrayOf (list of numbers; optional)
E   ?                        -----------
E   + - optionalArrayOf (list; optional)
``

Reply:
Yep, you made a clean improvement and we had locked down the wrong behavior with our tests. Fixed in https://github.com/plotly/dash/pull/726/commits/4deb9000f3c056ba1022772fb5479b3d4f647048